### DISCUSSION TITLE
Vehicle options not right

### DISCUSSION THREAD

**Discussion by ukoda** [tdorssers/TeslaPy#92] [General]
I was trying the demo cli.py and I noticed the options decoded don't match my vehicle in a few of significant ways.  For example it is reporting "Region: North America", "Left Hand Drive" and "Rear-Wheel Drive" where as I am in New Zealand with a RHD dual motor variant.  Looking at the option_codes string I see "DRLH" so I am assuming it is not the TeslaPy code at fault but the info supplied by Tesla.

**Comment by tdorssers** [comment]
Yes the option codes are deprecated by Tesla a few years ago. It always returns the same options for every tesla product. You can still decode the options of your vehicle if you got them from an order form or something. But you can't use this API to get them.
The 'vehicle_config' part is probably used by the official Tesla App and not the option codes.
